### ISSUE TITLE
Autocomplete with scandinavian characters

### ISSUE THREAD

**Issue by Perkyle** [JunoLab/Juno.jl#71]
I noticed an odd behavior:

In the console autocomplete with scandinavian characters, like ä and ö seems to work properly.
However, in the editor autocomplete fails with variables with such characters. However, if I define a variable with such characters in the console, then that variable autocompetes just fine in the editr - only the variables (etc) with such characters defined in the editor fail to autocomplete. Also, if I execute the variable defination (or function of whatever) in the editor, the autocomple will then work just fine.

**In summary**, if I create a variable (or whatever) with scandinavian characters in it in the editor and don't execute it, the autocomplete will fail.

**Comment by pfitzseb** [comment]
This is an issue with [autocomplete-plus](https://github.com/atom/autocomplete-plus)' default symbol provider. 
Long term solution is to disable that completely and use e.g. [this](https://github.com/pfitzseb/atom-julia-lsp-client) to provide local completions (i.e. completions of un-evaled or un-evalable code).

**Comment by pfitzseb** [comment]
Ah, found the issue: https://github.com/atom/autocomplete-plus/issues/76#issuecomment-187888937
You can enable that option at the bottom of the autocomplete-plus settings -- it works for me then.
